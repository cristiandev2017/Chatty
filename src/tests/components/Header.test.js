import { shallow } from "enzyme";
import Header from "../../components/Header";
import React from "react";

describe("Pruebas del componente <Header/>", () => {
  //Arrange
  const wrapper = shallow(<Header />);

  test("Comprobar que corra el componente", () => {
    //Como es solo para revisar que funcione y el componente no tiene acciones
    //El Assert sera un snapshot
    expect(wrapper).toMatchSnapshot();
  });

});