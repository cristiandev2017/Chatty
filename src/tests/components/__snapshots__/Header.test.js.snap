// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas del componente <Header/> Comprobar que corra el componente 1`] = `
<header>
  <nav
    className="navbar navbar-expand-sm fixed-top navbar-light bg-light"
  >
    <Link
      className="navbar-brand"
      to="/"
    >
      Chatty
    </Link>
    <button
      aria-controls="navbarNavAltMarkup"
      aria-expanded="false"
      aria-label="Toggle navigation"
      className="navbar-toggler"
      data-target="#navbarNavAltMarkup"
      data-toggle="collapse"
      type="button"
    >
      <span
        className="navbar-toggler-icon"
      />
    </button>
    <div
      className="collapse navbar-collapse justify-content-end"
      id="navbarNavAltMarkup"
    >
      <div
        className="navbar-nav"
      >
        <Link
          className="nav-item nav-link mr-3"
          to="/login"
        >
          Sign In
        </Link>
        <Link
          className="nav-item nav-link mr-3"
          to="/signup"
        >
          Sign Up
        </Link>
      </div>
    </div>
  </nav>
</header>
`;
